pkg/ctl/update/cluster.go

```diff
@@ -0,0 +1,99 @@
+package update
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/kris-nova/logger"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha4"
+	"github.com/weaveworks/eksctl/pkg/ctl/cmdutils"
+	"github.com/weaveworks/eksctl/pkg/eks"
+	"github.com/weaveworks/eksctl/pkg/printers"
+)
+
+var updateClusterDryRun = true
+
+func updateClusterCmd(g *cmdutils.Grouping) *cobra.Command {
```

Comment:
We could have a `--version` optional flag, when not specified it will use the latest stable version of EKS.

Reply:
This comment should go to #596 really. I agree we will probably need it, but not right away as right now there is only one version to upgrade to, but this code doesn't handle update yet anyway.